Makefile

```diff
@@ -16,6 +16,7 @@ BUILDFLAGS := -tags "$(BUILDTAGS)"
 BUILDAH := buildah
 SELINUXOPT ?= $(shell test -x /usr/sbin/selinuxenabled && selinuxenabled && echo -Z)
 SELINUXTYPE=container_runtime_exec_t
+AS ?= as
```

Comment:
Is there a platform we build on where this isn't provided by a default rule, as `LD` is?

Reply:
This is for the case where `as` command doesn't exist. Instead `x86-64-gnu-pc-linux-as` (or something similar) should be called directly.

Reply:
Honestly, you can drop from HEAD
```
AS ?= as
STRIP ?= strip
```
I added them to make sure to not break something accidentally

Reply:
`make -p -f /dev/null` with GNU make 4.4.1 on Fedora isn't showing me a predefined value for STRIP, so we'd need to keep that.